app/models/gsa18f/procurement.rb

```diff
@@ -26,14 +26,14 @@ def self.purchase_amount_column_name
     validates :product_name_and_description, presence: true
     validates :recurring_interval, presence: true, if: :recurring
 
-    after_create :add_approvals, :add_observers
-
-    def add_approvals
-      self.proposal.approver = User.for_email(Gsa18f::Procurement.approver_email)
-    end
-
-    def add_observers
-      self.add_observer(Gsa18f::Procurement.purchaser_email)
+    after_create :add_steps
```

Comment:
you know, in that case I would keep as a callback for now. If we have the factories doing things that our model doesn't do, that might get us into a situation where our tests aren't reflecting reality the way they should be.